src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/Whitelist.java

```diff
@@ -66,6 +66,70 @@
 
     public abstract boolean permitsStaticFieldSet(@Nonnull Field field, @CheckForNull Object value);
 
+    // Utility methods for creating canonical string representations of the signature
+    public static final StringBuilder joinWithSpaces(StringBuilder b, String[] types) {
```

Comment:
Can this be package-private or `@Restricted`? I do not see why users of this API plugin would call it. Ditto most of the other newly added members.

Reply:
Done